website/package.json

```diff
@@ -14,23 +14,24 @@
     "start": "docusaurus start",
     "build": "docusaurus build && yarn run update-redirect ./build/_redirects ./versions.json",
     "build:fast": "PREVIEW_DEPLOY=true yarn run build",
+    "tsc": "npx tsc --noEmit",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "serve": "docusaurus serve",
     "clear": "docusaurus clear",
     "test": "yarn build",
     "version:cut": "docusaurus docs:version",
-    "format:source": "prettier --write {{core,src}/**/*.js,*.js}",
+    "format:source": "prettier --write '{core,src}/**/*.{js,jsx,ts,tsx}'",
     "format:markdown": "prettier --write ../docs/*.md && prettier --write {versioned_docs/**/*.md,blog/*.md}",
     "format:style": "prettier --write src/**/*.{scss,css}",
     "format:examples": "eslint-examples-js --fix && eslint-examples-tsx --fix",
     "prettier": "yarn format:source && yarn format:markdown && yarn format:style",
-    "lint": "eslint ../docs/** blog/** core/** src/**/*.js ./*.js",
+    "lint": "eslint ../docs/** blog/** core/** src/**/*.tsx ./*.ts",
```

Comment:
Not that we should be encouraging anyone to write JS, but they could sneak it in at some point:
```suggestion
    "lint": "eslint ../docs/** blog/** '{core,src}/**/*.{js,jsx,ts,tsx}'",
```

Reply:
Prettier errors out, if there are no matching files in declarated paths:
![Screenshot 2024-12-03 at 23 03 24](https://github.com/user-attachments/assets/581802d6-af12-4be3-a1a8-bbec8333130e)

Reply:
That's if the matching paths are expanded (i.e. within brackets). If you keep it in string format, the pattern doesn't get expanded:
```
yarn run v1.22.22
$ prettier --write '{core,src}/**/*.{js,jsx,ts,tsx}'
core/DocsRating.js 34ms
core/PlatformTag.js 3ms
core/PrismTheme.js 6ms
core/TableRowWithCodeBlock.js 2ms
core/TabsConstants.js 6ms
core/VerticalTable.js 2ms
src/pages/Home/CallToAction/index.js 2ms
src/pages/Home/Community/index.js 5ms
src/pages/Home/components/Section/index.js 2ms
src/pages/Home/components/SectionTitle/index.js 2ms
src/pages/Home/components/ThemeImage/index.js 3ms
src/pages/Home/Framework/index.js 4ms
src/pages/Home/Hero/Devices.js 23ms
src/pages/Home/Hero/FloorBackground.js 1ms
src/pages/Home/Hero/GridBackground.js 2ms
src/pages/Home/Hero/index.js 2ms
src/pages/Home/index.js 1ms
src/pages/Home/Logo.js 1ms
src/pages/Home/Native/index.js 1ms
src/pages/Home/Platforms/FoxFact.js 3ms
src/pages/Home/Platforms/index.js 3ms
src/pages/Home/Watch/index.js 1ms
src/pages/index.js 1ms
src/pages/showcase.js 7ms
src/pages/versions.js 7ms
✨  Done in 0.48s.
```

Reply:
Nice, TIL!

Reply:
was wondering how i can suppress these errors, thanks!